packages/daimo-api/src/contract/coinIndexer.ts

```diff
@@ -132,6 +135,48 @@ export class CoinIndexer {
       txHash: transactionHash,
       logIndex,
       nonceMetadata,
+      opHash,
     };
   }
+
+  /* Attach fee amounts to transfer ops and filter out transfers involving
+   * paymaster.
+   */
+  private attachFeeAmounts(
+    transferOpsIncludingPaymaster: TransferOpEvent[]
+  ): TransferOpEvent[] {
+    // Map of opHash to fee amount paid to paymaster address
+    const opHashToFee = new Map<Hex, number>();
+    for (const op of transferOpsIncludingPaymaster) {
+      if (op.opHash === undefined) continue;
+
+      if (!opHashToFee.has(op.opHash)) {
+        opHashToFee.set(op.opHash, 0);
+      }
```

Comment:
```
const prevFee = opHashToFee.get(op.opHash) || 0;
```
then you can replace the ` opHashToFee.get(op.opHash)!` exprs below

Reply:
done